colpali_engine/models/florence2/colflor/configuration_florence2.py

```diff
@@ -0,0 +1,339 @@
+# ruff: noqa
+# coding=utf-8
+# Copyright 2024 Microsoft and the HuggingFace Inc. team. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.

```

Comment:
Looks this code is adapted `microsoft/Florence-2-large` from the Hf Hub. If yes, I would add the original link on top of the file. Moreover, I'm not a big fan of `# ruff: noqa` so I think we could get rid of it.

You might need to apply the ruff linter after this change!

```suggestion
# This code was copied and adapted from the "microsoft/Florence-2-large" model: https://huggingface.co/microsoft/Florence-2-large/blob/main/configuration_florence2.py

# coding=utf-8
# Copyright 2024 Microsoft and the HuggingFace Inc. team. All rights reserved.
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

```